packages/cbioportal-frontend-commons/src/components/downloadControls/DownloadControls.tsx

```diff
@@ -306,9 +311,7 @@ export default class DownloadControls extends React.Component<
     }
 
     render() {
-        if (
-            this.context.showDownloadControls === DownloadControlOption.HIDE_ALL
-        ) {
+        if (this.showDownload === false) {
```

Comment:
What's reason for eliminating context?

Reply:
the same reason detailed here https://github.com/cBioPortal/cbioportal-frontend/pull/4662